src/main/java/org/openlmis/stockmanagement/controller/StockCardController.java

```diff
@@ -94,7 +94,7 @@
 
     //TODO: Determine what the permissions associated with @PreAuthorize should be. (MANAGE_PROGRAM_PRODUCT, below, is just a placeholder).
 
-    @RequestMapping(value = "facilities/{facilityId}/products/{productId}/stockCard", method = GET, headers = ACCEPT_JSON)
+    @RequestMapping(value = "facilities/{facilityId}/products/{productCode}/stockCard", method = GET, headers = ACCEPT_JSON)
     @ApiOperation(value = "Get information about the stock card for the specified facility and product.",
```

Comment:
Please change the API documentation to reflect the change to the URL. (The documentation in the @ApiOperation section)